### DISCUSSION TITLE
no ResetDCW API

### DISCUSSION THREAD

**Discussion by sgf** [dahall/Vanara#437] [Q&A]
the .net inside is call ResetDCW not ResetDCA

**Comment by dahall** [comment]
`Gdi32.ResetDC` is set to use `CharSet.Auto` which means it will call `ResetDCA` on 32-bit systems and `ResetDCW` on 64-bit systems.

**Comment by sgf** [comment]
The ResetDCW function should be platform independent as this is just the wide character version of the API.
 I mean I didn't find the ResetDCW version of ResetDCA in Vanara. So when I called ResetDC, Vanara provided me with ResetDCA, which caused the application to report an error.

**Comment by sgf** [comment]
I have given up on using Vanara for now, I believe the API it provides is not comprehensive. I have switched to CSWin32 and imported the ResetDCW function myself.
